util/argconfig.c

```diff
@@ -389,11 +402,15 @@ int argconfig_parse(int argc, char *argv[], const char *program_desc,
 	long_opts[option_index++].val = 'h';
 
 	long_opts[option_index].name = "json";
-	long_opts[option_index].val = 'j';
+	long_opts[option_index++].val = 'j';
+
+	long_opts[option_index].name = "verbose";
+	long_opts[option_index].val = 'v';
```

Comment:
The commands need to support this. Just adding to the parser is not enough. Every single command needs to be updated so that also the help/usage output matches. Maybe there is way to have some default options.

Reply:
Fixed this as the default option mentioned.

Reply:
Cool! In the long run I would like to see getting the `-o/--output-format` option supported by all commands as well. So this is going into the right direction. Thanks!